### PR TITLE
Add contact form for mobile

### PR DESCRIPTION
**Pull request by Retky**
Add contact section with form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,153 +1,183 @@
...
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
-        </div>
+  <footer>
+    <!-- ↓CONCTACT↓ -->
+    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    <form id="form" action="index.html" method="post">
```
Comment (mahbubul14) [inline index.html:174]: - [ ] Please collect data by using the [Formspree service](https://formspree.io/html). Action value should be similar to `https://formspree.io/f/{form_id} `:+1:

File: index.html [hunk 2]
```diff
@@ -1,153 +1,183 @@
...
+      <input class="textInput" type="text" name="name" required placeholder="Full name" maxlength="30">
+      <input class="textInput" type="email" name="email" required placeholder="Email address">
+      <textarea id="message" name="name" maxlength="500" required placeholder="Enter text here"></textarea>
+      <button class="footerBtn" type="button" name="button" >Get in touch</button>
```
Comment (mahbubul14) [inline index.html]: - [x] I think it should be `type="submit"` for `button` element instead of `type="button"` :writing_hand:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
# Required Changes ♻️

Hi @Retky  @fasaldana  @mrjuda  , 

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

- [x] In your `readme` file, your live demo link is not updated. I couldn't see your contact form on that link. Please update the link properly.

- [x] Check the comments under the review to see the further changes. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Using Flexbox properly.

### Optional suggestions

- [ ] Your PR description should be a little bit descriptive to introduce changes in more detail.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (mahbubul14) [inline index.html]: - [x] Please collect data by using the [Formspree service](https://formspree.io/html). Action value should be similar to `https://formspree.io/f/{form_id} `:+1:

File: index.html (hunk 2 above)
Comment (Retky) [reply index.html]: You're right, the `type="button"` only executes thought a script.

**Comment by Retky** [comment]
Thank you @mahbubul14 
for the review. 😄

**Review by keroloussamy — APPROVED** [review]
Hi @Retky, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**